static/templates/settings/alert_word_settings.hbs

```diff
@@ -29,8 +29,8 @@
     </form>
     <div class="progressive-table-wrapper" data-simplebar>
         <table class="table table-condensed table-striped wrapped-table">
-            <thead>
-                <th class="active">{{t "Word" }}</th>
+            <thead class="alert-words-table">
+                <th data-sort="alphabetic" class="active">{{t "Word" }}</th>
```

Comment:
@yogesh-sirsat , I don't think it's appropriate to add data-sort attribute in this tag right now, as we are still to come up with functionality of sorting Alert words in alphabetical order, which is coming from #21143, which would solve add this change in background on hovering over alert-words table head.

Reply:
I just did as all tables `thead.th`, and as you can see hover method for this tables we use` &[data-sort]:hover`, but for alert words data-sort attr was missing that's why I just added this attribute, I know alert words sorting is not working for now, but isn't this attribute will useful later, If we removed data-sort attr then we would also have to change the method of hovering other tables `thead.th` or use separate CSS code for alert words.

Reply:
i know this would be useful later, but i don't think it's a good idea to add this right now with no other use of it except in applying CSS effect, but still i would leave this decision upto @timabbott ,

as i think this CSS effect for alert_words table would be fixed by the PR #21143, so need not to fix it here.

Reply:
Yeah, that's correct, we should only add `data-sort` for actual sorting logic,